### PR TITLE
OZ-34: Ensuring that ozone-frontend-config.json is loaded.

### PR DESCRIPTION
**Pull request by enyachoke**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -189,12 +189,12 @@ services:
     environment:
       SPA_PATH: /openmrs/spa
       API_URL: /openmrs
-      SPA_CONFIG_URLS:
+      SPA_CONFIG_URLS: /openmrs/spa/ozone/ozone-frontend-config.json
```
Comment (mks-d) [inline docker-compose.yml:192]: @enyachoke what does this actually _do_?

File: docker-compose.yml [hunk 2]
```diff
@@ -189,12 +189,12 @@ services:
...
     healthcheck:
       test: ["CMD", "curl", "-f", "http://localhost/"]
       timeout: 5s
-    # volumes:
-    #   - "${SPA_PATH:-microfrontends}:/usr/share/nginx/html/"
+    volumes:
+      - "${OZONE_CONFIG_PATH}/frontend:/usr/share/nginx/html/ozone"
```
Comment (mks-d) [inline docker-compose.yml:197]: @enyachoke is that related, or what is this change about?

**Review by mks-d — COMMENTED** [review]
I want to make the commit history meaningful, hence my questions. Looks like the headline of the PR doesn't describe what this commit does, even if this headline may be a side effect of the commit of course.

File: docker-compose.yml (hunk 1 above)
Comment (enyachoke) [reply docker-compose.yml:192]: The file itself is a general O3 frontend config file but in this case, it is actually adding the HIS component links. I got the message for the ticket but it looks okay to me.

File: docker-compose.yml (hunk 2 above)
Comment (enyachoke) [reply docker-compose.yml:197]: This is the volume that attaches the `ozone-frontend-config.json` file.

**Review by mks-d — APPROVED** [review]
